frontends/mit-open/src/pages/DashboardPage/SettingsPage.tsx

```diff
@@ -0,0 +1,131 @@
+import React from "react"
+import { PlainList, Typography, Link, styled } from "ol-components"
+import { useUserMe } from "api/hooks/user"
+import {
+  useSearchSubscriptionDelete,
+  useSearchSubscriptionList,
+} from "api/hooks/searchSubscription"
+
+const SOURCE_LABEL_DISPLAY = {
+  topic: "Topic",
+  unit: "MIT Unit",
+  department: "MIT Academic Department",
+  saved_search: "Saved Search",
+}
+
+const FollowList = styled(PlainList)(() => ({
+  borderRadius: "4px",
+  background: "#fff",
+  padding: "0px",
+}))
+
+const StyledLink = styled(Link)(({ theme }) => ({
+  color: theme.custom.colors.red,
+}))
+
+const TitleText = styled(Typography)(({ theme }) => ({
+  marginTop: "16px",
+  marginBottom: "8px",
+
+  color: theme.custom.colors.darkGray2,
+  ...theme.typography.h5,
+}))
+
+const SubTitleText = styled(Typography)(({ theme }) => ({
+  marginBottom: "16px",
+  color: theme.custom.colors.darkGray2,
+  ...theme.typography.body2,
+}))
+
+const ListItem = styled.li(({ theme }) => [
+  {
+    padding: "16px 32px",
+    display: "flex",
+    gap: "16px",
+    alignItems: "center",
+    border: `1px solid ${theme.custom.colors.lightGray2}`,
+    ":not(:last-child)": {
+      borderBottom: "none",
+    },
+    ":first-child, :last-child": {
+      borderRadius: "4px",
+    },
+  },
```

Comment:
Just use borderBottom for the "divider". Rest of border is on whole list.
```suggestion
    borderBottom: `1px solid ${theme.custom.colors.lightGray2}`,
    ":last-child": {
      borderBottom: "none",
    },
  },
```